app/models/merge_request.rb

```diff
@@ -2,4 +2,11 @@ class MergeRequest < ApplicationRecord
   belongs_to :requesting_organisation, class_name: "Organisation"
   has_many :merge_request_organisations
   has_many :merging_organisations, through: :merge_request_organisations, source: :merging_organisation
+  scope :unsubmitted, -> { where.not(status: "unsubmitted") }
```

Comment:
is this right? a scope of `unsubmitted` is where the `status` is not `unsubmitted`

Reply:
also by using an enum does it not create these scopes by default out of the box? or maybe i'm thinking of the predicates?

Reply:
Yup, you get that by default, in fact I was using it without realising 😬